zhaquirks/legrand/__init__.py

```diff
@@ -20,15 +43,74 @@ class LegrandCluster(CustomCluster):
     class AttributeDefs(BaseAttributeDefs):
         """Cluster attributes."""
 
-        dimmer = ZCLAttributeDef(
-            id=0x0000, type=t.data16, is_manufacturer_specific=True
+        device_mode = ZCLAttributeDef(
+            id=0x0000,
+            type=DeviceMode,
```

Comment:
This could be left like this, but do all devices actually accept all `DeviceMode` options?
Alternatively, we would only have different custom clusters per device (type) with matching enum options only.